sbb-connector-webtest/src_test/com/axonivy/connector/sbb/webtest/trip/ITBookTrip.java

```diff
@@ -0,0 +1,316 @@
+
+import com.axonivy.ivy.webtest.IvyWebTest;
+import com.axonivy.ivy.webtest.engine.EngineUrl;
+import com.codeborne.selenide.ElementsCollection;
+
+
+@IvyWebTest
+public class ITBookTrip {
+
+  private static final SimpleDateFormat DATE_FORMATTER_PRESENTABLE = new SimpleDateFormat("dd.MM.yyyy");
+  private static final SimpleDateFormat DATE_FORMATTER = new SimpleDateFormat("yyyy-MM-dd");
+  private static final SimpleDateFormat TIME_FORMATTER = new SimpleDateFormat("HH:mm");
+  private static final ZoneOffset SWISS_ZONE_OFFSET = ZoneId.of("Europe/Zurich").getRules().getOffset(Instant.now());
+
+  @Test
+  public void start_showInputTripSearchDataDialog_initialFieldValuesArePresent() {
+    open(EngineUrl.createProcessUrl("sbb-connector-demo/189FEADF3244D108/start.ivp"));
+    Date currentDate = new Date();
+
+    // Assert
+    $(By.id("form:from_input")).shouldHave(value(""));
+    $(By.id("form:to_input")).shouldHave(value(""));
+    $(By.id("form:date_input")).shouldHave(value(DATE_FORMATTER_PRESENTABLE.format(currentDate)));
+    $(By.id("form:time_input")).shouldHave(value(TIME_FORMATTER.format(currentDate)));
+    $(By.id("form:departureOrArrival_input")).shouldNotBe(checked);
+  }
+
+  @Test
+  public void start_inputTripSearchDataFieldsAreEmpty_proceedingShowsErrorMessages() {
+    open(EngineUrl.createProcessUrl("sbb-connector-demo/189FEADF3244D108/start.ivp"));
+
+    // Empty fields
+    $(By.id("form:from_input")).clear();
+    $(By.id("form:to_input")).clear();
+    $(By.id("form:date_input")).sendKeys(Keys.CONTROL, "A", Keys.DELETE);
+    $(By.id("form:time_input")).sendKeys(Keys.CONTROL, "A", Keys.DELETE);
+
+    // Proceed
+    $(By.id("form:proceed")).click();
+
+    // Assert
+    ElementsCollection errorMessages = $$(By.cssSelector(".ui-messages-error-summary"));
+
+    assertThat(errorMessages.size()).isEqualTo(4);
+    errorMessages.get(0).shouldHave(text("From: Value is required."));
+    errorMessages.get(1).shouldHave(text("To: Value is required."));
+    errorMessages.get(2).shouldHave(text("Date: Value is required."));
+    errorMessages.get(3).shouldHave(text("Time: Value is required."));
+  }
+
+  @Test
+  public void start_inputTripSearchDataFieldsDateAndTimeHaveInvalidContent_proceedingShowsErrorMessages() {
+    open(EngineUrl.createProcessUrl("sbb-connector-demo/189FEADF3244D108/start.ivp"));
+
+    // Insert invalid content into fields Date and Time
+    $(By.id("form:date_input")).sendKeys(Keys.CONTROL, "A", Keys.DELETE);
+    $(By.id("form:time_input")).sendKeys(Keys.CONTROL, "A", Keys.DELETE);
+
+    $(By.id("form:from_input")).setValue("A");
+    $(By.id("form:to_input")).setValue("B");
+    $(By.id("form:date_input")).setValue("C");
+    $(By.id("form:time_input")).setValue("D");
+
+    // Proceed
+    $(By.id("form:proceed")).click();
+
+    // Assert
+    ElementsCollection errorMessages = $$(By.cssSelector(".ui-messages-error-summary"));
+
+    assertThat(errorMessages.size()).isEqualTo(2);

```

Comment:
assertThat(errorMessage).hasSize(2)
-> will give you better error message if it fails